mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -172,6 +174,19 @@ def _expand_groups(groups: Iterable[str],
                 res.append(grp)
         return res
 
+    @staticmethod
+    def _expand_vars(params: Dict[str, TunableValue], global_config: Dict[str, TunableValue]) -> dict:
+        """
+        Expand `$var` into actual values of the variables.
+        """
+        res: Dict[str, TunableValue] = {}
+        for key, val in params.items():
+            if isinstance(val, str):
+                res[key] = Template(val).safe_substitute(global_config)
+            else:
+                res[key] = val
+        return res
```

Comment:
maybe something like
```suggestion
        return {
            key: Template(val).safe_substitute(global_config) if isinstance(val, str) else val
            for key, val in params.items()
        }
```